cli-feed-v4.json

```diff
@@ -41,12 +41,12 @@
       "hidden": false
     },
     "v4": {
-      "release": "4.101.0",
+      "release": "4.101.0", // Update to 102 after validation of prerelease first
       "releaseQuality": "GA",
       "hidden": false
     },
     "v4-prerelease": {
```

Comment:
these can move in lockstep. Validation can be done fully against the PR branch, and we don't need to go through the two-phase process